src/pages/studyView/charts/styles.module.scss

```diff
@@ -8,7 +8,7 @@
     background: white;
 
     &.highlight {
-        border: 2px solid #e5e076;
+        outline: 2px solid #e5e076;
```

Comment:
so if you notice currently in production, when you add a chart, there is a temporary yellow border on it, but border smushes all the inner contents inward which looks odd. when it came to this chart, it broke the axis tick placement until the border disappeared. so this just rewrites it so that the yellow border appears on the outside of the chart and doesnt disturb the actual chart